Testing System/src/main/java/com/epam/practice/testingsystem/data/dao/DbAccess.java

```diff
@@ -0,0 +1,47 @@
+package com.epam.practice.testingsystem.data.dao;
+
+import com.epam.practice.testingsystem.data.DataConnection;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+
+abstract class DbAccess {
+    protected static final String dbAccessExceptionMessage = "An error occurred while trying to access the database";
+
+    protected static int getLastInsertedId(Connection connection) throws SQLException {
+        try (PreparedStatement statement = connection.prepareStatement("SELECT LAST_INSERT_ID()")) {
+            ResultSet rs = statement.executeQuery();
+            rs.next();
+            return rs.getInt(1);
+        }
+    }
+
+    protected void delete1Arg(String table, String field, int id) {
+        String sql = String.format("DELETE FROM %s WHERE %s = ?", table, field);
+        try (Connection connection = DataConnection.getConnection()) {
+            PreparedStatement statement = connection.prepareStatement(sql);
+            statement.setInt(1, id);
+            statement.executeUpdate();
+        }
+        catch (SQLException e) {
+            e.printStackTrace();
+            throw new RuntimeException(dbAccessExceptionMessage);
+        }
+    }
+
+    protected void delete2Arg(String table, String field1, int id1, String field2, int id2) {
+        String sql = String.format("DELETE FROM %s WHERE %s = ? AND %s = ?", table, field1, field2);
```

Comment:
Maybe it will be better to rewrite the method by sending the Map with the structure like key=question, value=value. Then you can iterate through the map and delete as many values as you want.

The only problem is you will have to make your SQL statement is dynamic and you might not have enough time for doing such. So maybe the best way will be leaving it as is for now, but consider the above-mentioned approach for your future projects as a more clean and reliable one for production.